### ISSUE TITLE
Can Orgzly support Emacs diary entries?

### ISSUE THREAD

**Issue by alecvn** [orgzly/orgzly-android#856]
It is possible I am going the wrong way about this, so please advise if there is another way.

I'm currently importing my icalendar into my Emacs diary, as that is the simplest way to get ics files into Emacs (there's even a built-in `icalendar-import` function.

I would like to put my Emacs diary into my repository and have it populate my agenda in Orgzly alongside scheduled tasks and such.

I realise the diary file is not an Org file (different syntax), but I've seen some issues (#331, #269) discuss diary entries and just wanted to know if there's a way for me to have Orgzly pick those up, and if not, whether anyone has a workflow to get their icalendar into org-mode.

**Comment by ritschmaster** [comment]
I do not think that orgzly supports that. But if you are using your PC Emacs on the same org file, then you might be happy with the following: https://www.reddit.com/r/emacs/comments/ho8lxi/mu4e_office365_meeting_invitations/

So the idea is to actually import the iCalendar invitations to org which will then be synced to your phone.
